src/utils/helperFuncs.js

```diff
@@ -0,0 +1,5 @@
+export const getFirstChar = (str = '') => str.charAt(0).toUpperCase();
+
+export const isFollowing = (user, authUserId) => {
+  return user?.followers?.find((followers) => followers === authUserId) || false;
```

Comment:
**Suggestion:** use some method instead of find to only return true if found otherwise false